README.md

```diff
@@ -29,25 +33,29 @@ If you've already cloned the repo, run this command inside of the repository:
 ## Follow the build steps
 
 ### Windows
-[Install vcpkg](https://github.com/microsoft/vcpkg#quick-start-windows), then run the following:
-- `[vcpkg root]\vcpkg.exe install libtheora libogg --triplet=x64-windows-static` (the triplet can be whatever preferred)
+To handle dependencies, you'll need to install [Visual Studio Community](https://visualstudio.microsoft.com/downloads/) (make sure to install the `Desktop development with C++` package during the installation) and [vcpkg](https://github.com/microsoft/vcpkg#quick-start-windows).
+
+After installing those, run the following in Command Prompt (make sure to replace `[vcpkg root]` with the path to the vcpkg installation!):
+- `[vcpkg root]/vcpkg.exe install libtheora libogg --triplet=x64-windows-static` (If you're compiling a 32-bit build, replace `x64-windows-static` with `x86-windows-static`.)
 
-Finally, follow the [compilation steps below](#compiling) using `-DCMAKE_TOOLCHAIN_FILE=[vcpkg root]/scripts/buildsystems/vcpkg.cmake -DVCPKG_TARGET_TRIPLET=(chosen triplet)` as arguments for `cmake ..`.
+Finally, follow the [compilation steps below](#compiling) using `-DCMAKE_TOOLCHAIN_FILE=[vcpkg root]/scripts/buildsystems/vcpkg.cmake -DVCPKG_TARGET_TRIPLET=x64-windows-static` as arguments for `cmake -B build`.
```

Comment:
Maybe use something similar to `(chosen triplet)` to remove the extra bullet point below? If you think this is clearer still then keep it as is.

Reply:
My thinking with how it is currently is that this will minimize the amount of things that the user has to modify in the command; we just assume the user is on x64, and if they're not, the bullet point explains how to set it up for x86. Since the average user will have no idea what a triplet is, I think it might be best to leave a default like this.